homeassistant/components/walkingpad/__init__.py

```diff
@@ -0,0 +1,500 @@
+"""Support for Xiaomi WalkingPad treadmill."""
+from __future__ import annotations
+
+import asyncio
+import logging
+from typing import Any, Final
+
+from bleak.exc import BleakError
+from miio.exceptions import DeviceException
+from miio.walkingpad import OperationMode, Walkingpad, WalkingpadStatus
+from ph4_walkingpad.pad import Controller, WalkingPad
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_IP_ADDRESS, CONF_TOKEN, STATE_OFF, STATE_ON
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo, Entity
+
+from .const import (
+    AWAIT_SLEEP_INTERVAL,
+    CONF_CONN_TYPE,
+    CONF_DEFAULT_SPEED,
+    CONF_UUID,
+    DEFAULT_SPEED,
+    DOMAIN,
+    MAX_SPEED,
+    MIN_SPEED,
+    MODES_DICT,
+)
+
+PARALLEL_UPDATES: Final = 1
+
+PLATFORMS: Final = ["remote"]
```

Comment:
Please use the new enumerator:
```suggestion
PLATFORMS: Final = [Platform.REMOTE]
```